src/components/ArmyDetailsCard.vue

```diff
@@ -0,0 +1,182 @@
+<template >
+   <div class="flip-card" >
+       <div class="flip-card-inner">
+           <div class="flip-card-front" :style={background:color}>
+               <img :src="imgurl" alt=""/>
+            </div>
+            <div class="flip-card-back">
+                 <h2>{{armydata.Kind}}</h2>
+                 <h6>{{armydata.Description}}</h6>
+                 <div className='traits'>
+                    <ul>
+                        <li>Strength:{{armydata.Strength}}</li>
+                        <li>Agility:{{armydata.Agility}}</li>
+                        <li>Intellignce:{{armydata.Intelligence}}</li>
+                    </ul>
+                    <ul >
+                        <li v-for="(tarrian) in tarrians" v-bind:key="tarrian">{{tarrian}}</li>
+                    </ul>
+                 </div>
+                </div>
+            </div>
+   </div>
+</template>
+<script>
+export default {
+
+  name: 'armycarddetails',
+  props: ['armydata', 'color'],
```

Comment:
use lowerCamelCase for naming a variable

Reply:
bro here you  mean props name should be something like
armyData